manage.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This is a command management for Robottelo including shell command
+
+    $ python manage.py shell
+    Welcome to Robottelo Interactive shell
+        Auto imported: [settings, entities, rt, ng]
+    In [1]: org = entities.Organization().create()
+    ... DEBUG: ... making POST resquest to...
+
+Depends on click:
+    pip install click
+"""
+import click
+import nailgun
+import code
+import readline
+import rlcompleter
+import importlib
+from robottelo.config import settings
+
+
+@click.group()
+def core_cmd():
+    """ Core commands wrapper """
+    pass
```

Comment:
pass is not needed when the function have a docstring, not a blocker, just mentioning